lib/src/main/kotlin/com/swmansion/starknet/data/types/Responses.kt

```diff
@@ -156,6 +156,7 @@ data class BlockWithTransactionsResponse(
     override val sequencerAddress: Felt,
 
     // Block body
+    // TODO (after reviews): move above block_header to align with other similar responses
```

Comment:
remove once moved.

Reply:
let's just do it :+1: it may be beneficial to include all the necessary changes in reviews.

Reply:
Ok, but it diff would make it hard to review.